### PR TITLE
[WIP] L7 no connectivity in direct routing mode

### PR DESCRIPTION
**Pull request by brb**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/8632)

### CODE REVIEW

**Comment by brb** [comment]
test-me-please

**Comment by brb** [comment]
test-me-please

**Comment by brb** [comment]
test-me-please

**Comment by brb** [comment]
test-me-please

**Comment by brb** [comment]
test-me-please

File: test/k8sT/manifests/v1.5/cilium-ds-patch.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@ spec:
         - "--pprof=true"
         - "--log-system-load"
         - "--config-dir=/tmp/cilium/config-map"
+        - "--debug-verbose=flow"
```
Comment (joestringer) [inline test/k8sT/manifests/v1.5/cilium-ds-patch.yaml:18]: Does this work with the helm stuff now? Also, this is the manifest for v1.5 but AFAICT you're not doing the upgrade tests so it should be running the dev/master version of the test..?

Comment (brb) [reply test/k8sT/manifests/v1.5/cilium-ds-patch.yaml:18]: Please ignore this patch leftover.

**Pull request closed without merging**